src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -3035,6 +3042,7 @@ export default defineComponent({
       valueChangeIcon.value = icon
       showValueChangePopup.value = true
       invertValueChangeContentOrder.value = invertContentOrder
+      showOverlayControls()
```

Comment:
This makes `showOverlayControls` called twice in `seekBySeconds`
Maybe just call `showOverlayControls` under some of the `showValueChange()` calls?